assets/scss/modules/_buttons.scss

```diff
@@ -3,14 +3,19 @@
     display: inline-block;
     text-align: center;
     text-decoration: none;
-    text-transform: uppercase;
     border: none;
     outline: none;
-    margin: 0 5px;
     appearance: none;
     font-family: $dp-font-family-proximanova;
     border-radius: $dp-border-radius;
-    @include transition;
+    @include transition-all;
+
+    .ms-icon {
+      font-size: 20px;
+      line-height: 7px;
+      position: relative;
+      left: -5px;
+    }
 
     &.button-big {
```

Comment:
Se está usando BEM como nomenclatura de clases? Si ese es el caso, esta clase debería tener `--` (doble guión medio) ya que es una clase modificadora del padre.

Reply:
Ok Damian

Reply:
@mvillaseco do you have any opinion on this?

Reply:
Este punto es importante para aclarar, como dice Damian si la nomenclatura es BEM puede ir de la siguiente forma '--' (doble guión medio), '_' simple guión bajo o '-' simple guión medio que es lo que se viene utilizando. 

1) button button--big button--big__active
2) button button_big  button_big--active
3) button button-big  button-big__active

Cual es la mejor opción a utilizar ?

Reply:
Hasta puede ser de la siguiente forma: dp-button dp-button--small

Reply:
Por convención lo usamos como aclaro Dami arriba.